_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+To save valuable mobile data, turning off video thumbnails and profile pictures for comments was already possible. Now even the black placeholders which replaced them are gone, and comments can comfortably take up the space left over. Thanks, [@4D17Y4](https://github.com/4D17Y4)!
 
-[@TobiGr](https://github.com/TobiGr) brought two more improvements into this release: the setting to remember pop-up size and position has a less redundant description, and when you switch to another app, the main player now minimizes to background audio by default to allow for a more seamless transition. Swoosh.
+[@TobiGr](https://github.com/TobiGr) brought two more improvements: The setting to remember pop-up size and position has a less redundant description, and when switching to another app, the main player now minimizes to play audio in the background by default. Seamless transition. Swoosh.
 
-It's 2020, and our default resolutions also realized this. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS now, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded the last time, and will auto-select it when you hit download on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
+It is the current year, 2020, in higher resolution. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded last time, and will auto-select it when you hit "Download" on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
 
-Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for any confusion.
+Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for confusion.
 
 ### More Translation updates and improvements
 
-Apart from the language improvements mentioned above, some other interesting localization changes happened in this release. [@comradekingu](https://github.com/comradekingu), a frequent contributor to and language checker of NewPipe and other open source projects, has once again taken a close look at our strings. There was, as always, a lot of room for improvement. Our wonderful translators rushed to help and immediately implemented his suggestions.
```

Comment:
Localization isn't interesting. Language improvements aren't interesting. I am not interesting. I don't work on "open source", and that isn't how that is written. I work on libre software. I am not suggesting, I am changing the source language. The reader isn't involved in the process, and that is a shame, because rarely do you see any localization in the changelog.

Reply:
> Localization isn't interesting. Language improvements aren't interesting. I am not interesting.

TobiGr thought your contributions are important and wanted to mention them. We agree. You might not want the credit but you will have it either way. Deal with it. 😎

> I don't work on "open source".... I work on libre software. 

"and other open source **projects**"

But sure, next time we'll say "libre software projects". Good?

> I am not suggesting, I am changing the source language.

Yeah, that slipped in at the last moment. Should have been changed.

> The reader isn't involved in the process, and that is a shame, because rarely do you see any localization in the changelog. Opportunity missed.

That's why we need you. ;)

Reply:
@opusforlife2 
No, not good. Bad.
I don't want to be part of this blogpost. It is bad, and it misrepresents me.
For whatever I do here, I am done. You need your egos.